.github/workflows/pr-title-commitlint-check.yml

```diff
@@ -1,11 +1,12 @@
-name: PR Title Commitlint Check
+name: Commitlint
```

Comment:
I'd keep "PR" in the name, to make it clear it's a non-standard usage of Commitlint (it's not actually linting any commits).